sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting/tests/PlaywrightServiceSettingsTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Identity;
+using Azure.Developer.MicrosoftPlaywrightTesting;
+using System;
+
+namespace Azure.Developer.MicrosoftPlaywrightTesting.Tests;
+
+[TestFixture]
+public class PlaywrightServiceSettingsTest
```

Comment:
The tests running against Azure resources must use the [Azure.Core Test Framework](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework) - specifically its `TestEnvironment` and the Azure SDK  [test resources](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/README.md) system.  Both are heavily tied into our pipelines and release process.

Reply:
These are just unit tests for our SDK. None of these are running against any Azure resources.

Reply:
Have added live tests